polymetis/polymetis/torch_isolation/src/torch_server_ops.cpp

```diff
@@ -86,6 +128,7 @@ TorchScriptedController::~TorchScriptedController() {
 }
 
 std::vector<float> TorchScriptedController::forward(TorchRobotState &input) {
+  torch::NoGradGuard no_grad;
```

Comment:
Did you see any performance improvements on setting `no_grad`?

Reply:
It just straight up crashes in PyTorch 1.10 without `torch::NoGradGuard`. Comparing benchmarks under `Run tests` between [this branch](https://app.circleci.com/pipelines/github/facebookresearch/fairo/3550/workflows/0154a75b-6910-4418-a020-f5facb679abb/jobs/13360) and [main](https://app.circleci.com/pipelines/github/facebookresearch/fairo/3535/workflows/80744876-7ac5-435c-bd6f-2be3f51209b4/jobs/13261), I don't see significant differences (it seems a tad faster, but CircleCI benchmarking variance is pretty high).